src/components/common/markets/dashboard-group.tsx

```
@@ -0,0 +1,135 @@
+
+interface DashboardGroupProps {
+  type: DashboardGroupType
+  categoryName: string
+  markets: Market[]
+}
+
+export const DashboardGroup = ({ markets, type, categoryName }: DashboardGroupProps) => {
+  const showCardLayout = (type: DashboardGroupType) => {
+    switch (type) {
+      case 'row':
+        const rowMarkets = markets.slice(0, 2)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)